metrics/memory/analyze_heap_memory.py

```
@@ -0,0 +1,968 @@
+    path: Path
+    kind: str
+    ts: Optional[datetime]
+    mtime: float
+
+    @property
+    def id(self) -> str:
+        return self.path.name
+
+    @property
+    def sort_time(self) -> datetime:
+        return self.ts if self.ts else datetime.fromtimestamp(self.mtime)
```

Ruff (0.15.1)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)